pallets/pallet-dip-consumer/Cargo.toml

```diff
@@ -53,3 +53,5 @@ std = [
   "frame-benchmarking?/std",
   "sp-runtime?/std",
 ]
+
+try-runtime = []
```

Comment:
`frame-system` and other pallets are missing. Can be done in another PR I guess. Just afraid that the try-runtime tests are not working right now. Did you verify that?

Reply:
The pallet does not have any support for `try-runtime` yet. It was required for clippy not to complain. I guess once we add support for `try-runtime` (e.g., once we have a migration), then we will import the required dependencies?

Reply:
I mean for the runtime to compile, but in essence having the feature enabled has no effect on those pallets (yet).